addons/source-python/data/source-python/entities/csgo/CBasePlayer.ini

```diff
@@ -58,7 +58,7 @@ srv_check = False
         offset_windows = 372
 
 
-[instance_attribute]
+[instance_property]
```

Comment:
Maybe we should call it ``based_attribute`` or ``based_property`` as you can specify whether it is an ``instance_attribute`` or ``pointer_attribute`` using the ``method`` key.

I think I would prefer ``based_attribute``.

Reply:
> 
> 
> Maybe we should call it `based_attribute` or `based_property` as you can specify whether it is an `instance_attribute` or `pointer_attribute` using the `method` key.
> 
> I think I would prefer `based_attribute`.

Alright, I've updated the code.